pages/common/hangups.md

```diff
@@ -0,0 +1,15 @@
+# hangups 
+
+> Third party client for Google Hangouts.
+
+- Start hangups:
+
+`hangups`
+
+- View troubeshooting information:
```

Comment:
I am guessing this shows the help page. In that case, its better described as "View the help page."